salt/orch/update-etc-hosts.sls

```diff
@@ -0,0 +1,23 @@
+update_grains:
+  salt.function:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - name: saltutil.refresh_grains
```

Comment:
According to [this](https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.saltutil.html#salt.modules.saltutil.refresh_grains), _"New in version 2016.3.6,2016.11.4,Nitrogen."_. Doesn't it mean that is available since _2016.3.6_?